docs/sources/datasources/elasticsearch.md

```diff
@@ -103,6 +109,15 @@ Each data link configuration consists of:
 The Elasticsearch query editor allows you to select multiple metrics and group by multiple terms or filters. Use the plus and minus icons to the right to add/remove
 metrics or group by clauses. Some metrics and group by clauses haves options, click the option text to expand the row to view and edit metric or group by options.
 
+## PPL Query editor
+
+The Elasticsearch query editor allows you to write PPL queries if PPL support is enabled. Select PPL in the dropdown menu next to the query input field to switch to the PPL query editor.
+PPL queries can be written in the query input field.
```

Comment:
```suggestion

Write PPL queries in the query input field.

```

Reply:
Try to avoid sentences in passive voice. Active voice is clearer and uses fewer words.